app/i18n/ja/people.ts

```diff
@@ -177,6 +194,7 @@ const people: People = {
           'MOSH副会長、取締役会メンバー',
           'ノースフロリダ大学コンピューティング理事会メンバー',
         ],
+        link: 'https://www.linkedin.com/in/richardnall/?originalSubdomain=uk',
```

Comment:
I think you meant to use https://www.linkedin.com/in/khwajashaik/? Double check the other `/people.ts` files as well.

Reply:
done